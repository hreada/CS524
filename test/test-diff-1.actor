letactor(bind(dec, lambda(m, minus(m, 1))),
         bind(a  , lambda(m, send(dec, 42))),
         send(a, nil))
