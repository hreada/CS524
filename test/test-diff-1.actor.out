<T>
  <actors>
    <actor>
      <k> exec ( nil ) </k>
      <id> initactor </id>
    </actor>
    <actor>
      <k> exec ( nil ) </k>
      <id> _2 </id>
    </actor>
    <actor>
      <k> exec ( 41 ) </k>
      <id> _1 </id>
    </actor>
  </actors>
  <messages> .MessageCellBag </messages>
  <definedaddr> ListItem ( initactor ) ListItem ( _1 ) ListItem ( _2 ) </definedaddr>
</T>
